09-features.md

```diff
@@ -36,6 +36,7 @@ The following `globalfeatures` bits are currently assigned by this specification
 | Bits | Name              | Description                                                        | Link                                  |
 |------|-------------------|--------------------------------------------------------------------|---------------------------------------|
 | 8/9  | `var_onion_optin` | This node requires/supports variable-length routing onion payloads | [Routing Onion Specification][bolt04] |
+| 14/15| `option_anchor_outputs` | Anchor outputs                                               | [BOLT #3](03-transactions.md)         |
```

Comment:
nit: name could be `option_bring_your_fees` or `option_dynamic_fee_adjustement` to encompass changes on HTLC txn and underscores the use of CPFP.

Reply:
Yes, something to think about. There are still fees attached to the txes, but possibly only minimal. The real changes are the anchor outputs plus the htlc sighash change. Dynamic fee adjustment sounds like something that we already have with `update_fee`.

Reply:
`option_just_in_time_fees`?, naming is hard..

Reply:
Yes, it is hard. The fundamental decision is probably whether we describe what the option does (`option_anchor_outputs`), what it enables (`option_bring_your_own_fees`) or why you want it (`option_always_confirm`).

We keep the brainstorm open...